xbmc/cores/VideoPlayer/DVDDemuxers/DVDDemuxFFmpeg.cpp

```diff
@@ -1461,6 +1461,20 @@ CDemuxStream* CDVDDemuxFFmpeg::AddStream(int streamIdx)
     stream->flags = (CDemuxStream::EFlags)pStream->disposition;
 
     AVDictionaryEntry *langTag = av_dict_get(pStream->metadata, "language", NULL, 0);
+    if (!langTag)
+    {
+      if ((strcmp(m_pFormatContext->iformat->name, "avi") == 0) && (streamIdx > 0 && streamIdx < 10))
```

Comment:
No, sorry I did not find any documentation, but some source code.

Source code of VLC Media Player ([avi.c](https://github.com/videolan/vlc/blob/master/modules/demux/avi/avi.c)):
At [line 392](https://github.com/videolan/vlc/blob/master/modules/demux/avi/avi.c#L392) it is started to traverse all streams and search for information.
At [line 426](https://github.com/videolan/vlc/blob/master/modules/demux/avi/avi.c#L426) audio streams are sorted out and [from line 482 to 489](https://github.com/videolan/vlc/blob/master/modules/demux/avi/avi.c#L482-L489) the language information are retrieven.
To calculate the RIFF tag for the corresponding stream they use the formula "IAS(1 + (currentStreamIndex - 1))" (currentStreamIndex starting with the value 0, which references to stream 0) (at line [485](https://github.com/videolan/vlc/blob/master/modules/demux/avi/avi.c#L485)).

This means that IAS1 corresponds to stream 1, IAS2 corresponds to stream 2 ...

This would result in the following changes:
- use the normal streamIdx from CDVDDemuxFFmpeg::AddStream
- skip all non-audio streams
- skip stream 0, because IAS0 is not defined (?)